test/server/http_dynamic_delay_filter_integration_test.cc

```diff
@@ -0,0 +1,186 @@
+#include <string>
+
+#include "external/envoy/test/integration/http_integration.h"
+
+#include "api/server/response_options.pb.h"
+
+#include "server/common.h"
+#include "server/http_dynamic_delay_filter.h"
+
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+using namespace testing;
```

Comment:
I bet you can remove this. If you do need to use something from `testing` and you don't want to qualify it in place, you can put the more specific using directive(s) here instead.

e.g. `using testing::HasSubstr;`